Source/WebInspectorUI/Localizations/en.lproj/localizedStrings.js

```diff
@@ -156,6 +156,10 @@ localizedStrings["All Changes"] = "All Changes";
 localizedStrings["All Events @ Event Breakpoint"] = "All Events";
 /* Break (pause) on all exceptions */
 localizedStrings["All Exceptions @ JavaScript Breakpoint"] = "All Exceptions";
+/* Font Details Panel */
+localizedStrings["No Custom Fonts @ Font Details Sidebar Property"] = "No Custom Fonts";
```

Comment:
please do not manually edit this file.  it's autogenerated by running
```
./Tools/Scripts/extract-localizable-js-strings --utf8 ./Source/WebInspectorUI/Localizations/en.lproj/localizedStrings.js ./Source/WebInspectorUI/UserInterface
```

Reply:
I ran the command and got:
```/* Message shown when there are no custom fonts. */
localizedStrings["No Custom Fonts @ Font Details Sidebar Property"] = "No Custom Fonts";```
populated.